### PR TITLE
fixed messages for not defined variables

### PR DESCRIPTION
**Pull request by omalave**
I created a simple fix for those not defined variables, that prevent to "php compact function" throw notices and warnings.

### CODE REVIEW

**Review by ChristianOrralaBB — APPROVED** [review]
Ok, just setting the value to null, this should prevent the warnings and notices.

**Comment by thefrosty** [comment]
I'd like to test this before you merge. Please allow me a day to get this tested locally.

**Review by thefrosty — CHANGES_REQUESTED** [review]
Needs to be tested locally before approval.

**Review by thefrosty — APPROVED** [review]

**Pull request merged**
